sympy/series/tests/test_series.py

```diff
@@ -202,6 +202,13 @@ def test_issue_9549():
     assert series(y, x, oo) == x**(-5) - 1/x**4 + x**(-3) + 1/x + O(x**(-6), (x, oo))
 
 
+def test_issue_9173():
+    var('p_0 p_1 p_2 p_3 b_0 b_1 b_2')
```

Comment:
You are not storing them in variables which you are using, that's why test is failing.

Reply:
Thanks for guidance @sidhu1012